content/docs/communities/switching-providers/_index.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Switching Providers"
+weight = 150
+[extra]
+emoji = "🏡"
+tile = "I want to get my own homeserver"
+last_updated = "2022-10-27T08:00:00Z"
++++
+
+* Why?
+    * Branding
+    * Recovering accounts and/or rooms
+    * Staying in control of where your data is
+    * Adding more, less crowded bridges
+* How?
+    * Relying on a [hosting provider](/ecosystem/hosting)
+    * Hosting one of [the server implementations](/ecosystem/servers) yourself
```

Comment:
```suggestion
- Why?
    - Branding
    - Recovering accounts and/or rooms
    - Staying in control of where your data is
    - Adding more, less crowded bridges
- How?
    - Relying on a [hosting provider](/ecosystem/hosting)
    - Hosting one of [the server implementations](/ecosystem/servers) yourself
```

Reply:
Those are just notes for now, which were laid out mostly to give structure to the section. I'll flesh things out and get rid of the bullet points.